models/migrations/v112.go

```diff
@@ -30,8 +28,7 @@ func removeAttachmentMissedRepo(x *xorm.Engine) error {
 		}
 
 		for i := 0; i < len(attachments); i++ {
-			uuid := attachments[i].UUID
-			if err = os.RemoveAll(path.Join(setting.AttachmentPath, uuid[0:1], uuid[1:2], uuid)); err != nil {
+			if err = storage.Attachments.Delete(relativePath(attachments[i].UUID)); err != nil {
```

Comment:
This will run before attachments are actually migrated from the local store - this should use the local store variant.